scripts/gen_self_signed_cert.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```

Comment:
>python [](start = 15, length = 6)

to lay it on heavy, should this be "gen_self_signed_cert_for_testing_do_not_use_for_production.py"?